extensions/interactions/TextInput/TextInputPredictionServiceSpec.js

```diff
@@ -47,6 +47,7 @@ describe('Text Input Prediction Service', function() {
         for(var j = 0; j < trainingData[i].answers.length; j++) {
           predictedAnswerGroup = predictionService.predict(
             classifierData, trainingData[i].answers[j]);
+          // eslint-disable-next-line eqeqeq  
           if(predictedAnswerGroup == trainingData[i].answer_group_index) {
```

Comment:
Btw, can you put a space before '(' here? (And also on line 47?)

Might be something to try and add a linter rule for as well :)

Reply:
Sorry, but I didn't quite get you. Should I put a space here: `if (..` or towards the end bracket? Thanks!

Reply:
First one. The end bracket is not a '('.

I.e., should be `if (...`

Reply:
Sorry, my bad. I'll make the necessary changes. Thanks!

Reply:
Fixed, thanks!